project/cmake/installdata/addon-bindings.txt

```diff
@@ -31,3 +32,6 @@ xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_inputstream.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_peripheral.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_pvr.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_codec.h
+xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxPacket.h
+xbmc/cores/AudioEngine/Utils/AEChannelData.h
+xbmc/filesystem/IFileTypes.h
```

Comment:
I think `kodi_vfs_utils.hpp` was forgotten, but was there special reason about the other three or also forgotten?